libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -101,6 +101,27 @@ void CodeTransform::operator()(FunctionCall const& _call)
 	solAssert(m_scope, "");
 
 	m_assembly.setSourceLocation(_call.location);
+
+	if (_call.functionName.name == "abort")
```

Comment:
This could be handled with a vector which lists the name, argument and return value count.